content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -1,132 +1,113 @@
-### The Concept of OOP in Java: Getting Started with Abstraction and Encapsulation
- 
-Object-Oriented Programming (OOP) is a programming paradigm (approach), as is procedural programming. Unlike Procedural programming, we model everything as objects (Real-life entities). Java, as an object-oriented programming language, was design to follow this pattern.
- 
-Objects come from class, a form of generalization, but a real thing is an object. What then is a class?
- 
-Class is a template for objects creation.
  
-The process of creating objects from a class is instantiation.
+### The Concept of OOP in Java: Getting Started with Abstraction and Encapsulation
  
-In this tutorial, you will learn about abstraction, encapsulation, objects, and classes.
+Object-Oriented Programming (OOP) is a programming paradigm (approach), as is procedural programming. Unlike procedural programming, we model everything as objects (real-life entities). Java, as an object-oriented programming language, was designed to follow this pattern.
  
-To say the least, the concept of OOP will be a nice piece to grab for absolute java beginners.
+Objects come from a class, a form of generalization, but a real thing is an object. What exactly is a class? A class is a template to create objects.
+The process of creating objects from a class is called instantiation.
  
+In this tutorial, you will learn about abstraction, encapsulation, objects, and classes. The concept of OOP will be a nice piece to grab for absolute Java beginners.
+
 ### Prerequisites
- 
 The reader should have:
- 
 - At least a basic knowledge of Java
- 
 - Have Java installed on your computer
- 
 - Have any IDE of your choice
  
 ### Objectives
- 
 At the end of this first OOP tutorial series, readers should be able to:
- 
 - Differentiate between classes and objects
- 
 - Understand abstraction and encapsulation
- 
 - Create objects from classes
- 
 - Have a working knowledge of what OOP is about
- 
 - Create abstract classes and interfaces
  
-### Class:
+### Class
  
 A class is a template or a blueprint for object creation.
+To represent attributes or properties, classes use instance variables whereas, for behaviors or actions, it uses instance methods.
  
-To represent attributes or properties, a class uses instance variables. For behaviors or actions, it uses instance methods.
+### Instance Variables
```

Comment:
For the two subheaders *Instance Variables* and *Instance Methods*, add an extra \# so that each use four \#s (i.e. \#\#\#\# Instance Variables and \#\#\#\# Instance Methods). This way, we can denote that they belong under the section about classes.